src/plib/controllers/IndexController.php

```diff
@@ -127,8 +130,62 @@ private function getUsageList($currentPath, array $usage)
         ]);
 
         $list->setData($data);
+
+        if (!empty($data)) {
+            $listTools = [
+                [
+                    'title' => \pm_Locale::lmsg('buttonDelete'),
+                    'execGroupOperation' => [
+                        'skipConfirmation' => false,
+                        'subtype' => 'delete',
+                        'locale' => ['confirmOnGroupOperation' => \pm_Locale::lmsg('confirmDelete')],
+                        'url' => $this->_helper->url('delete-selected'),
+                    ],
+                    'class' => 'sb-delete-selected',
+                ],
+            ];
+
+            $list->setTools($listTools);
+        }
+
         $list->setDataUrl($this->_helper->url('index-data', 'index', null, ['path' => $currentPath]));
 
         return $list;
     }
+
+    public function deleteSelectedAction()
+    {
+        $paths = (array) $this->_getParam('ids');
+        $serverFileManager = new \pm_ServerFileManager;
```

Comment:
You don't need the backslashes in this class, you are already in the global namespace.